src/component/header/Header.tsx

```diff
@@ -107,12 +107,7 @@ function HeaderInner(props: HeaderInnerProps) {
   }, [height, modal]);
 
   const openAboutUs = useCallback(() => {
-    modal.show(<AboutUsModal />, {
-      isBackgroundBlur: false,
-      position: positions.MIDDLE,
-      width: 500,
-      height: 480,
-    });
+    modal.show(<AboutUsModal />);
   }, [modal]);
```

Comment:
You shouldn't use the `modal.show` API anymore. Have a look at the stories and the code in `react-science`. Basically you need a state with `useOnOff` and render the modal component in the children.

Reply:
fixed thank you